src/options/mod.rs

```diff
@@ -123,6 +125,8 @@ pub struct Options {
 
     /// Whether to read file names from stdin instead of the command-line
     pub stdin: FilesInput,
+
+    pub archive_inspection: ArchiveInspection,
```

Comment:
I think that archive Inspection belong in the `view` part of options, as it modifies the output by adding informations. And we have all of those types of options in it.

Reply:
I can move it to `view`, although I'm not sure if I agree that the flag only modifies the output since (as mentioned above in this issue) it also acts as a feature gate which might modify the access time of files.
But if no one else has an opinion on the matter, I'll simply move it to `view`.

Reply:
well even if in view you can feature gate it, exemple is git